Code/Framework/AzCore/AzCore/Math/Matrix3x4.inl

```diff
@@ -203,6 +203,17 @@ namespace AZ
     }
 
 
+    AZ_MATH_INLINE Matrix3x4 Matrix3x4::CreateFromMatrix4x4(const Matrix4x4& matrix4x4)
+    {
+        Matrix3x4 result;
```

Comment:
Why do you use column and not rows?
Unless I'm mistaken, matrices are built of rows and so, for getting columns the performance penalty seem to be quite high?!